pages/level/[id]/[slugName].tsx

```diff
@@ -59,23 +59,18 @@ export async function getStaticPaths() {
 }
 
 interface LevelParams extends ParsedUrlQuery {
-  id: string;
+  params: string[];
```

Comment:
Tried running this locally and I get errors here (params is undefined). The only way I've gotten it to work so far is by changing it to:
```
id: string;
slugName: string;
```